fe/fe-core/src/main/java/org/apache/doris/fs/obj/ObjStorage.java

```diff
@@ -32,6 +32,8 @@
  * @param <C> cloud SDK Client
  */
 public interface ObjStorage<C> {
+    public static final int CHUNK_SIZE = 5 * 1024 * 1024;
```

Comment:
add comment what is chunk_size and what it is for

Reply:
> add comment what is chunk_size and what it is for

done